app/models/comment.php

```diff
@@ -44,4 +47,60 @@ public function write()
         $db = DB::conn();
         $db->insert('comment',$params);
     }
+
+    public static function getByThreadId($thread_id)
+    {
+        $db = DB::conn();
+        return $db->row(
+            sprintf("SELECT * FROM comment WHERE thread_id=%d LIMIT %d", $thread_id, self::FIRST_COMMENT)
```

Comment:
This would be vulnerable to sql injection because thread_id was not passed as parameter to the prepared statement :bow: